### PR TITLE
Adds audio for when you are hurt

### PR DESCRIPTION
**Pull request by Qustinnus**
This is pretty barebones right now, we might want to consider adding some kind of chance/cooldown/whatever to this, and also I need to replace the sound because right now it's probably not what we want :^)

### CODE REVIEW

**Comment by unusualcrow** [comment]
oof

File: Content.Server/GameObjects/Components/Mobs/SpeciesComponent.cs [hunk 1]
```diff
@@ -6,17 +6,22 @@
...
+using Robust.Shared.IoC;
+using Robust.Server.GameObjects.EntitySystems;
 
 namespace Content.Server.GameObjects
 {
     [RegisterComponent]
     public class SpeciesComponent : SharedSpeciesComponent, IActionBlocker, IOnDamageBehavior, IExAct
     {
+
+        [Dependency] private readonly IEntitySystemManager _entitySystemManager;
```
Comment (Zumorica) [inline Content.Server/GameObjects/Components/Mobs/SpeciesComponent.cs]: You should probably disable warning CS0649 (Field never assigned to) when using IoC dependencies.
Like this:
```csharp
#pragma warning disable 649
    [Dependency] private readonly IEntitySystemManager _entitySystemManager;
#pragma warning restore 649
```

**Comment by PJB3005** [comment]
Not sure this is a good idea. At least not this bare-bones implementation:

1. Sound can't be different based on context, so you'd *oof* from oxyloss/toxloss too.
2. There's already hit sounds for stuff like melee weapons.
3. (Alright man I'm definitely not merging it with the Minecraft sound)

**Comment by Qustinnus** [comment]
kek well i wasnt gonna keep the audio. do we pass damage type on this?

**Comment by Zumorica** [comment]
DamageEventArgs specifies DamageType

**Comment by SweptWasTaken** [comment]
Contacted floyd on discord and it doesn't look like he's gonna update this anytime soon.

**Pull request closed without merging**